tests/libnet/cloudinit.go

```diff
@@ -197,3 +197,19 @@ func CreateDefaultCloudInitNetworkData() (string, error) {
 		),
 	)
 }
+
+// CreateDynamicIPv6CloudInitNetworkData generates a configuration for the
+// Cloud-Init Network Data, in version 2 format.
+// The configuration sets dynamic IPv4 (DHCP) and IPv6 (stateful dhcpv6)
+// addresses, inclusing DNS settings of the cluster nameserver IP and search
+// domains.
+func CreateDynamicIPv6CloudInitNetworkData() (string, error) {
```

Comment:
Please explain why do we introduce a new method and not updating `CreateDefaultCloudInitNetworkData`?

Reply:
`CreateDefaultCloudInitNetworkData` is used also for (at least) sr-iov: https://github.com/kubevirt/kubevirt/blob/727dec427340b79cd8a76b94f2f7b0393e916c99/tests/vmi_multus_test.go#L1363

I am not interested in fixing that as part of this pr.

Reply:
Why not? The sriov vmi has fedora os and its primary interface has masquerade binding.
Isn't that the goal of the commit (fedora vmis with primary masquerade interface to use stateful dhcpv6)?

You can see my [commit](https://github.com/kubevirt/kubevirt/pull/4555/commits/425a405124950177e2478ad6d61b75e7e119cc4f) in the union closed PR that does exactly what I suggested, changes the already existing `CreateDefaultCloudInitNetworkData` and all the tests using it passed smoothly.